crawlers/d-crawler/.gitignore

```diff
@@ -0,0 +1,14 @@
+.dub
```

Comment:
um.... d-crawler is not currently used at all.

don't want to throw this out though